eq-author/src/App/section/Design/SectionEditor/HubSettings/index.js

```diff
@@ -38,65 +38,70 @@ const EnableDisableWrapper = styled.div`
 `;
 
 const HubSettings = ({ id, requiredCompleted, showOnHub }) => {
-
   const [updateSection] = useMutation(updateSectionMutation);
   const defaultOpen = () => requiredCompleted;
 
   return (
     <Collapsible
-        title="Hub settings"
-        className="hubSettings"
-        defaultOpen={defaultOpen()}
-        withoutHideThis
-        variant="content"
+      title="Hub settings"
+      className="hubSettings"
+      defaultOpen={defaultOpen()}
```

Comment:
I know you didn't do this, but this being a function is unecessary so would be good to swap it around since you're here.

```suggestion
      defaultOpen={requiredCompleted}
```

Then you can get rid of the function definition.